packages/calcite-components/src/tests/utils.ts

```diff
@@ -150,6 +150,28 @@ export async function getElementXY(
   );
 }
 
+/**
+ * Helper to get an E2EElement's DOMRect object.
```

Comment:
Awesome! I think this should replace `getElementXY`. Can you mark add a deprecation note to it?